notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>Move H1 heading to first line in next links cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done